packages/connect-ui/src/lib/query.ts

```diff
@@ -0,0 +1,17 @@
+import { QueryClient } from '@tanstack/react-query';
+
+// Create a client
+export const queryClient = new QueryClient({
+    defaultOptions: {
+        queries: {
+            refetchInterval: 0,
+            refetchOnWindowFocus: false,
+            refetchOnMount: true,
+            staleTime: 60 * 1000 * 5,
+            retry: 0,
+            retryDelay: (attemptIndex) => {
+                return Math.min(2000 * 2 ** attemptIndex, 30000);
+            }
```

Comment:
do you need do define retryDelay if retry = 0?

Reply:
no but I might reenable retry at some point, it was annoying now because the backend is mostly broken :p